CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to GenRMQ
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to GenRMQ. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+As we are a fairly small community so far, we will keep these guidelines very lightweight for now.
+
+## How to Contribute
+
+We happily accept contributions as [GitHub PRs][github_prs] or bug reports, comments/suggestions or usage questions by creating a [GitHub issue][gen_rmq_issues].
+
+## Code of Conduct
+
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
In situations like this, I personally prefer to omit the list of characteristics given that it is impossible to enumerate all the characteristics for which someone may be self-conscious. Perhaps a slightly modified version of the Golden Rule where we just pledge to treat everyone with respect?

```
We as members, contributors, and leaders pledge to make participation in our community a pleasant experience for everyone. All are welcome to participate and can expect to be treated with politeness and respect. Likewise, we expect participants to treat other members, contributors, leaders and participants with politeness and respect. Harassment for any reason based on immutable or mutable characteristics will not be tolerated.
```

Reply:
Creating a complete list of such characteristics certainly seems hard. However I find them useful as examples. I took this introduction from the [Contributor Covenant v2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct/), assuming that they thought about this longer and harder than I could.

For the paragraph that you are quoting above, were did you get that from?

Reply:
@akoutmos I think we can actually park this conversation for now, as I realized only now that we already have a Code of Conduct in a separate file. 

So rather than changing the existing Code Of Conduct, I would like to focus this PR on introducing a first small version of our Contribution Guidelines.